mla/models.py

```diff
@@ -112,12 +113,8 @@ def _create_background_dec_spline(self, sin_dec_bins: np.array, *args,
         # behavior, we first take a look at the dec values
         # in the data. We can do this by histogramming them.
         sin_dec = np.sin(self._data['dec'])
-
-        # Make the background histogram. Note that we do NOT
-        # want to use density=True here, since that would mean
-        # that our spline depends on the original bin widths!
-        weights = np.ones_like(self._data['dec'])/len(self._data['dec'])
-        hist, bins = np.histogram(sin_dec, bins=sin_dec_bins, weights=weights)
+        hist, bins = np.histogram(sin_dec, bins=sin_dec_bins, density=True)
+        
```

Comment:
did we determine that this is the correct change?

Reply:
Yes. I checked with my old code and skylab.